releasenotes/notes/mcx_recursive_clean_ancilla-2b0f6956e0f4cbbd.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    :class:`.MCXRecursive` with k controls and a single clean auxiliary qubits now requires at most
+    16*k-8 cx gates.
```

Comment:
```suggestion
    :class:`.MCXRecursive` with :math:`k` controls and a single clean auxiliary qubits now requires at most
    :math:`16k-8` CX gates.
```

Reply:
Typo: it should be "a single ... qubit" (not "qubits"). Also, for consistency, it might be better to day "with k control qubits" instead of "with k controls".